src/Sylius/Component/Core/OrderProcessing/OrderTaxesProcessor.php

```
@@ -77,15 +88,19 @@ public function process(BaseOrderInterface $order): void
         }
 
         $zone = $this->getTaxZone($order);
-
         if (null === $zone) {
             return;
         }
 
+        $customerTaxCategory = $this->getCustomerTaxCategory($order);
+        if (null === $customerTaxCategory) {
+            return;
```

We should probably throw an exception here.